cmd/config.go

```diff
@@ -24,6 +24,16 @@ import (
 	"github.com/minio/mc/pkg/console"
 )
```

Comment:
This file name can be change to be more appropriate, `config.go` is means it holds `config` data structure - these are just fileutil functions for config.